cpu/sam0_common/periph/rtc.c

```diff
@@ -41,13 +43,45 @@ static rtc_state_t rtc_callback;
 * Thanks to this, the user will be able to set time in 2000's*/
 static uint16_t reference_year = 100;
 
-void rtc_init(void)
+static void _wait_syncbusy(void)
+{
+#ifdef REG_RTC_MODE2_SYNCBUSY
+    while (RTC->MODE2.SYNCBUSY.reg);
```

Comment:
please use empty braces `while(foo) {}` for each while.

Reply:
alright!